modules/dnn/test/test_tf_importer.cpp

```diff
@@ -146,6 +146,7 @@ TEST_P(Test_TensorFlow_layers, padding)
     runTensorFlowNet("padding_valid");
     runTensorFlowNet("spatial_padding");
     runTensorFlowNet("keras_pad_concat");
+    runTensorFlowNet("mirror_pad");
```

Comment:
Please open PR into [opencv_extra](https://github.com/opencv/opencv_extra) with test data (use the same source branch name as this PR: `ocv_mirrorpad`)

Reply:
add pull request([622](https://github.com/opencv/opencv_extra/pull/622)) to opencv_extra